src/aiida/engine/processes/calcjobs/manager.py

```diff
@@ -87,7 +88,7 @@ def last_updated(self) -> Optional[float]:
         """
         return self._last_updated
 
-    async def _get_jobs_from_scheduler(self) -> Dict[Hashable, 'JobInfo']:
+    async def _get_jobs_from_scheduler(self) -> Dict[str, 'JobInfo']:
```

Comment:
optional, since you're already touching the type:

```suggestion
    async def _get_jobs_from_scheduler(self) -> dict[str, 'JobInfo']:
```

Reply:
moved to PR #7114